contracts/modules/capital/Pool.sol

```diff
@@ -191,15 +188,15 @@ contract Pool is IPool, MasterAware, ReentrancyGuard {
 
   }
 
-  function setAssetDetails(
+  function setSwapDetails(
     address assetAddress,
     uint104 _min,
     uint104 _max,
     uint16 _maxSlippageRatio
   ) external onlyGovernance {
 
     require(_min <= _max, "Pool: min > max");
-    require(_maxSlippageRatio <= 1 ether, "Pool: Max slippage ratio > 1");
+    require(_maxSlippageRatio <= 100, "Pool: Max slippage ratio > 1");
```

Comment:
to confirm 1 is 1% right?

Reply:
1 is 0.01%

Reply:
The original check was to make sure `_maxSlippageRatio` did not exceed 100% (not 1%). This would mean the check should be `_maxSlippageRatio <= 10000` (not 100).